indy_common/authorize/authorizer.py

```diff
@@ -73,11 +79,17 @@ def authorize(self,
         if is_role_accepted is None:
             return False, "sender's DID {} is not found in the Ledger".format(request.identifier)
         if not is_role_accepted:
-            return False, "role is not accepted"
+            return False, "{} can not do this action".format(self.get_named_role_from_req(request))
         if not self.is_sig_count_accepted(request, auth_constraint):
-            return False, "count of signatures is not accepted"
+            return False, "Not enough signatures"
         if not self.is_owner_accepted(auth_constraint, auth_action):
-            return False, "actor must be owner"
+            if auth_action.txn_type == NYM:
```

Comment:
Maybe we should rather check if this is `AuthActionEdit` action and get the field from there (action.field) instead of having hard-coded `NYM` and `verkey`?